rozdzial3.tex

```diff
@@ -10,7 +10,7 @@ \section{Liczba}
 
 \section{Rysunek}
 
-Pakiet \texttt{subcaption} pozwala na umieszczanie w podpisie rysunku odnośników do ,,podilustracji'': \\
+Pakiet \texttt{subcaption} pozwala na umieszczanie w podpisie rysunku odnośników do ``podilustracji'': \\
```

Comment:
` ``podilustracji'' ` stworzy angielski cudzysłów, który nie występuje w j. polskim. `,,podilustracji''` poprawnie stworzy polski cudzysłów.

Reply:
To powoduje następujące ostrzeżenie:

> You ought to remove spaces in front of punctuation. (26)

Przy okazji znalazłem [wytłumaczenia ostrzeżeń](http://www.nongnu.org/chktex/ChkTeX.pdf)

Reply:
Jak możesz zobaczyć na [wiki](https://pl.wikibooks.org/wiki/LaTeX/Formatowanie#Cudzys%C5%82owy) `,,''` to poprawny sposób wprowadzania polskich cudzysłowów więc najwyraźniej ChkTeX się myli sądząc że to dwa przecinki po sobie.